pkg/common/token/store.go

```diff
@@ -59,6 +60,12 @@ func (b *BasicTokenStore) GetType() string {
 
 // GetToken implements TokenStore.
 func (b *BasicTokenStore) GetToken(ctx context.Context) (*Token, error) {
+	// This ensures lastToken not to be nil while b.lastTokenRefreshError != nil.
```

Comment:
```suggestion
	// This ensures lastToken not to be nil while b.lastTokenRefreshError == nil.
```
Or maybe my understanding is wrong